doc/sphinx/source/projects/ad41101_iio/ad41101_iio.rst

```diff
@@ -0,0 +1,106 @@
+
+**Supported Evaluation Boards:**
+
+* `EVAL-AD4110-1 <https://www.analog.com/en/resources/evaluation-hardware-and-software/evaluation-boards-kits/eval-ad4110-1.html>`_
+
+**Supported Carrier Boards:**
+
+* `SDP-K1 With Mbed <https://os.mbed.com/platforms/SDP_K1/>`_
+
+============
+Introduction
+============
+
+This page gives an overview of using the ARM platforms supported (default is Mbed) 
+firmware example with Analog Devices AD4110 Evaluation board and SDP-K1 controller board. 
+This example code leverages the ADI developed IIO (Industrial Input Output) ecosystem to 
+evaluate the AD4110 device by providing a device debug and data capture support.
+
+   .. image:: /source/tinyiiod/app_interface.png
+      :width: 350
+
+The interface used for communicating with PC based IIO clients is either Virtual Serial Or UART. 
+IIO Firmware leverages the ADI created no-os and platform driver software layers
+to communicates with IIO device.
+
+.. SDP-K1 Mbed Section
+
+.. include:: /source/tinyiiod/sdp_k1_mbed.rst
+
+.. Useful links Section
+
+.. include:: /source/useful_links.rst
+
+====================
+Hardware Connections
+====================
+
+Required: SDP-K1 (or alternative Mbed enabled controller board), EVAL-AD4110 board
+and USB cable.
+
+Connect the EVAL-AD4110 board to SDP-K1 board (or any other Mbed enabled controller 
+board) using jumper wires. Connect SDP-K1 board to the PC using the USB cable. 
+
+===============
+Jumper Settings
+===============
+
+**SDP-K1:**
+
+Connect the VIO_ADJUST jumper on the SDP-K1 board to 3.3V position to drive SDP-K1 GPIOs at 3.3V
+
+**EVAL-AD4110:**
+
+* Stack the EVAL-AD4110-SDZ on the Arduino connectors of the SDP-K1 board.
+* Please refer to the EVAL-AD4110-SDZ user guide to know the jumper connections to the EVAL board
+
+====================================
+Quick Start to use Mbed IIO Firmware
+====================================
+If you have some familiarity with the Mbed platform, the following is a basic list of steps required to start running the code, see below for more detail:
+
+* Connect the AD4110 EVAL-board to the SDP-K1 controller board as specified in hardware connections section.
+* Connect the SDP-K1 controller board to your computer over USB provided along with SDP-K1 board.
+* Go to the link of the code provided above in the 'Downloads' section and import code into Keil studio.
+* Ensure SDP-K1 controller board is selected.
+* Set the imported project folder as active project (Right click on the ad717x_mbed_iio-application→ Set Active Project).
+* Make sure that all the dependent libraries are imported. This happens automatically when the active project has been configured.
+* Compile the code.
+* After a successful compile a binary will be downloaded to your computer - store this on your drive.
+* Drag and drop this binary to the USB drive hosted by your controller board.
+
+   .. image:: /source/projects/ad41101_iio/sdp_k1_target_keil.png
```

Comment:
Is this pic needed if this is already being covered in the Build section above?

Reply:
This image is not present in the build guide, it was present on wiki and hence I have added the same

Reply:
I meant a similar pic was already part of the build section. Isn't this duplicated?